src/main/java/graphql/execution/FieldValueInfo.java

```diff
@@ -66,7 +67,7 @@ public String toString() {
     public static class Builder {
         private CompleteValueType completeValueType;
         private CompletableFuture<Object> fieldValueFuture;
-        private List<FieldValueInfo> listInfos = new ArrayList<>();
+        private List<FieldValueInfo> listInfos = ImmutableList.of();
```

Comment:
I have another PR that intends to break this class away from a builder pattern

https://github.com/graphql-java/graphql-java/pull/3465

eg JUST allocate the the object - not a builder and then the object say

Reply:
I was also going to add a `SingletonMergedField` but `MergedField` is concrete and marked as a public API, so didn't feel comfortable changing that.